db/migrate/20251118045164_add_default_discount_code_id_to_links.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddDefaultDiscountCodeIdToLinks < ActiveRecord::Migration[7.1]
+  def change
+    add_column :links, :default_discount_code_id, :integer, null: true, default: nil
```

Comment:
Please switch to `bigint`

We can potentially remove `null: true, default: nil` since that's the default?